tests/clpy_tests/core_tests/test_ndarray_complex_ops.py

```diff
@@ -59,12 +59,12 @@ def test_real_setter(self, xp, dtype):
         x.real = testing.shaped_reverse_arange((2, 3), xp, dtype).real
         return x
 
-    @testing.for_dtypes('FD')
-    @testing.numpy_clpy_array_almost_equal(accept_error=False)
-    def test_imag_setter(self, xp, dtype):
-        x = testing.shaped_arange((2, 3), xp, dtype)
-        x.imag = testing.shaped_reverse_arange((2, 3), xp, dtype).real
-        return x
+#    @testing.for_dtypes('FD')
```

Comment:
Please leave a comment about the reasons why this is ignored

Reply:
https://github.com/fixstars/clpy/pull/170#issuecomment-461686197

I mean leaving it on the code, not just on this PR.

Reply:
The reason for comment out is that ClPY does not correspond to complex numbers, and there is no need to perform tests to handle imaginary parts.

Reply:
I understand, I will add the reason for commenting out the code.

Reply:
I added comment on the code.